src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -235,6 +248,19 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    onIconClick() {
+        this.dropdownCourses = !this.dropdownCourses;
+    }
+
+    /** Listen click event whether on outside of the container to close the dropdown menu */
+    @HostListener('document:click', ['$event'])
```

Comment:
ngbDropdown has a input called `autoClose`. If set to true the dropdown will close on inside and outside clicks.  (https://ng-bootstrap.github.io/#/components/dropdown/api)

Reply:
done